packages/uikit-workshop/penthouse.js

```diff
@@ -0,0 +1,66 @@
+const fs = require('fs');
+const path = require('path');
+const penthouse = require('penthouse');
+
+function buildCriticalCSS(cb) {
+  penthouse({
+    url: 'file:///' + __dirname + '/dist/index.html', // @todo: need to figure out dynamic port if serving page itself
+
+    // the original css to extract critcial css from
+    css: path.resolve(process.cwd(), 'dist/styleguide/css/pattern-lab.css'),
+
+    // OPTIONAL params
+    width: 1300, // viewport width
+    height: 900, // viewport height
+
+    // when true, will not filter out larger media queries
+    keepLargerMediaQueries: true,
+
+    // selectors to keep
+    forceInclude: [
+      // '.my-selector',
+      '.pl-c-body--theme-light',
```

Comment:
should all the theme classes be considered? or are they picked up by already being the default? That would make sense

Reply:
The light theme was the one variation Penthouse wasn’t immediately picking up when calculating out the Critical CSS for the page.

Ideally we’d just have these config options be exported in one of our JS files and the build step here pulls those in automatically (similar to how we deal with a ton of these gotchas in the Bolt Design System code).

Basically, if we go in and update which theme options are available, our build process stays up to date!